pkg/controller/kibana/driver.go

```diff
@@ -87,7 +91,45 @@ func secretWatchFinalizer(kibana kbtype.Kibana, watches watches.DynamicWatches)
 	}
 }
 
-func (d *driver) deploymentParams(kb *kbtype.Kibana) (deployment.Params, error) {
+func (d *driver) getStrategyType(kb *kbtype.Kibana) (appsv1.DeploymentStrategyType, bool, error) {
```

Comment:
Maybe a comment explaining the return types specifically the meaning of `bool`. It is clear at the call site but without context harder to understand maybe.